app/__init__.py

```diff
@@ -4,6 +4,6 @@
 app = Flask(__name__)
 app.secret_key = "tomibomi"
 
-
+from app.logs.logger import logger
```

Comment:
bad practise imo, you import something in 2 steps just to save writing 2 words

Reply:
done